yelpapi/yelpapi.py

```diff
@@ -60,9 +60,12 @@ class YelpAPIError(Exception):
         """
         pass
 
-    def __init__(self, client_id, client_secret):
-        self._yelp_session = OAuth2Session(client=BackendApplicationClient(client_id=client_id))
-        self._yelp_session.fetch_token(token_url=ACCESS_TOKEN_URL, client_id=client_id, client_secret=client_secret)
+    def __init__(self, api_key):
+        ''' Instantiate a YelpAPI object.
```

Comment:
Would you mind formatting these docstrings like the other docstrings in this project? That is, use double-quotes, space/indent as seen elsewhere.